cmd/analyze.go

```diff
@@ -56,15 +56,34 @@ func checkAnalyzeArgNum(args []string) error {
 	return nil
 }
 
-func analyzeImage(imageArg string, analyzerArgs []string) error {
+func analyzeImage(imageName string, analyzerArgs []string) error {
 	cli, err := NewClient()
 	if err != nil {
 		return fmt.Errorf("Error getting docker client for differ: %s", err)
 	}
 	defer cli.Close()
+
+	analyzeTypes, err := differs.GetAnalyzers(analyzerArgs)
+	if err != nil {
+		glog.Error(err.Error())
+		return errors.New("Could not perform image analysis")
```

Comment:
Could you wrap the previous error instead of making a new one? No need to log there either.